Sources/SteamPress/Provider.swift

```diff
@@ -21,7 +21,7 @@ public struct Provider: Vapor.Provider {
         let viewFactory = LeafViewFactory(viewRenderer: drop.view)
 
         // Set up the controllers
-        let blogController = BlogController(drop: drop, pathCreator: pathCreator, viewFactory: viewFactory, postsPerPage: postsPerPage, config: drop.config)
+        let blogController = BlogController(drop: drop, pathCreator: pathCreator, viewFactory: viewFactory, postsPerPage: postsPerPage, enableAuthorsPages: drop.config.enableAuthorsPages, enableTagsPages: drop.config.enableTagsPages, config: drop.config)
```

Comment:
Almost there! Haha, what I was thinking was something similar to the `useBootstrap4` configuration. So you would pull it out of the configuration for the initialiser with the `Config`, or pass it in to the main initialiser. Make sense?